module_evaluations/TCR-30_2023-09-20.MD

```diff
@@ -0,0 +1,66 @@
+  * This depends on `mod-fqm-manager` for which `edge-fqm` is the edge module of and the `mod-fqm-manager` module is not yet approved as of this review.
+* [x] Module gracefully handles the absence of third party systems or related configuration
+* [x] Sonarqube hasn't identified any security issues, major code smells or excessive (>3%) duplication
+  * No major issues reported by Sonarqube, but Sonarcube is also reporting "No Coverage information (61.9% Estimated after merge)".
+* [x] Uses [officially supported](https://wiki.folio.org/display/TC/Officially+Supported+Technologies) build tools
+  * This uses unit tests related dependencies not on the supported technologies list, such as com.squareup.okhttp3.mockwebserver.
+  * The list of technologies does not go into details on such dependencies.
+* [x] Unit tests have 80% coverage or greater, and are based on [officially approved technologies](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)
+  * ~The Coverage is only at 61.9% according to Sonarcube.~
+  * As of commit hash [adae385241c928f79c9723a5f7039ade3fefc903](https://github.com/folio-org/edge-fqm/commit/adae385241c928f79c9723a5f7039ade3fefc903) the test coverage reported is 93.7%.
+  * This uses test related dependencies not on the supported technologies list, such as com.squareup.okhttp3.mockwebserver.
+
+## Frontend
+* [x] ~If provided, End-to-end tests must be written in an [officially approved technology](https://wiki.folio.org/display/TC/Officially+Supported+Technologies)~
+* [x] ~Have i18n support via react-intl and an `en.json` file with English texts~
+* [x] ~Have WCAG 2.1 AA compliance as measured by a current major version of axe DevTools Chrome Extension~
+* [x] ~Use the latest release of Stripes at the time of evaluation~
+* [x] ~Follow relevant existing UI layouts, patterns and norms~
+* [x] ~Must work in the latest version of Chrome (the supported runtime environment) at the time of evaluation~
+
+## Backend
+* [x] Module's repository includes a compliant Module Descriptor
+* [x] ~Module includes executable implementations of all endpoints in the provides section of the Module Descriptor~
+* [x] Environment vars are documented in the ModuleDescriptor
+* [x] ~If a module provides interfaces intended to be consumed by other FOLIO Modules, they must be defined in the Module Descriptor "provides" section~
+* [ ] All API endpoints are documented in RAML or OpenAPI
+  * Only `/entity-types` is showing up in the build-time generated API and SDK Documentation.
+  * The following endpoints are missing: `/query`.
+  * The `edge-fqm.yaml` does not have references to `/query` but `@FeignClient` is used and references the `/query` endpoints in `QueryClient` and `QueryController` (using `@RestCintroller`).
```

Comment:
Worth noting: this module mostly shares a common API with mod-fqm-manager, so their OpenAPI spec is mostly defined here: https://github.com/folio-org/folio-query-tool-metadata/tree/master/src/main/resources/swagger.api

We'll need to figure out what we need to do to make that show up in the generated documentation, since the shared library works for generating the necessary interfaces and object schemas 😟